### PR TITLE
fix(islamic-website): optimize images

### PR DESCRIPTION
**Pull request by hadzhehsen**
closes: HQW-127

### CODE REVIEW

File: libs/islamic-website/index-page/src/lib/index-page/index-page.tsx [hunk 1]
```diff
@@ -29,6 +29,8 @@ export function IndexPage({
   mainnetAccounts: number;
   locale: string;
 }) {
+  console.log({ executiveMembers });
```
Comment (olegshilov) [inline libs/islamic-website/index-page/src/lib/index-page/index-page.tsx]: revert

**Review by olegshilov — CHANGES_REQUESTED** [review]

File: libs/islamic-website/index-page/src/lib/index-page/index-page.tsx (hunk 1 above)
Comment (hadzhehsen) [reply libs/islamic-website/index-page/src/lib/index-page/index-page.tsx]: Done

File: libs/islamic-website/index-page/src/lib/funds-block/funds-block.tsx [hunk 2]
```diff
@@ -51,7 +51,12 @@ export function FundsBlock() {
               aria-label="Optic Capital"
             >
               <div className="relative h-[80px] w-[100px] opacity-60 transition-opacity duration-300 hover:opacity-100">
-                <Image src={opticCapitalLogoImgData.src} alt="" fill />
+                <Image
+                  src={opticCapitalLogoImgData.src}
+                  alt=""
```
Comment (olegshilov) [inline libs/islamic-website/index-page/src/lib/funds-block/funds-block.tsx]: set alt for all fund logos like this
```suggestion
                  alt="Optic Capital"
```

File: libs/islamic-website/index-page/src/lib/funds-block/funds-block.tsx [hunk 3]
```diff
@@ -51,7 +51,12 @@ export function FundsBlock() {
               aria-label="Optic Capital"
             >
               <div className="relative h-[80px] w-[100px] opacity-60 transition-opacity duration-300 hover:opacity-100">
-                <Image src={opticCapitalLogoImgData.src} alt="" fill />
+                <Image
+                  src={opticCapitalLogoImgData.src}
```
Comment (olegshilov) [inline libs/islamic-website/index-page/src/lib/funds-block/funds-block.tsx]: better to use image data as src prop for next images
```suggestion
                  src={opticCapitalLogoImgData}
```

**Review by olegshilov — CHANGES_REQUESTED** [review]

File: libs/islamic-website/index-page/src/lib/funds-block/funds-block.tsx (hunk 2 above)
Comment (hadzhehsen) [reply libs/islamic-website/index-page/src/lib/funds-block/funds-block.tsx]: Done

File: libs/islamic-website/index-page/src/lib/funds-block/funds-block.tsx (hunk 3 above)
Comment (hadzhehsen) [reply libs/islamic-website/index-page/src/lib/funds-block/funds-block.tsx]: Done

**Review by olegshilov — APPROVED** [review]

**Pull request merged**